src/main.rs

```diff
@@ -326,17 +299,36 @@ fn main() {
     env_logger::init().expect("Can't instantiate env logger");
     println!("semantic.rs 🚀");
 
-    let args: Args = Docopt::new(USAGE)
-        .and_then(|d| d.decode())
-        .unwrap_or_else(|e| e.exit());
-
-
-    if args.flag_version {
-        println!("semantic.rs 🚀 -- v{}", VERSION);
-        process::exit(0);
-    }
 
-    let config = assemble_configuration(args);
+    let clap_args =  App::new("semantic-rs")
+        .version(VERSION)
+        .author("Jan Schulte <hello@unexpected-code> & Jan-Erik Rediger <janerik@fnordig.de>")
+        .about("Crate publishing done right")
+        .arg(Arg::with_name("write")
+             .short("w")
+             .long("write")
+             .help("Write changes to files (default: yes if CI is set, otherwise no).")
+             .takes_value(false))
+        .arg(Arg::with_name("release")
+            .short("r")
+            .long("release")
+            .help("Create release on GitHub and publish on crates.io (only in write mode) [default: yes].")
```

Comment:
I think with above code `release` mode is only really enabled if this flag is passed, right?
Also given that it's now a flag without any argument labeling it to have a default seems wrong